example-site/components/templates/Content.stories.tsx

```diff
@@ -0,0 +1,14 @@
+import { PageTemplate } from '../../../.storybook/components/PageTemplate';
+import { Content as ContentComponent } from './Content';
+
+export default {
+	title: 'Templates/Content',
+};
+
+export function Content() {
```

Comment:
Could you also export a story where no sidebar is shown? (like our updates page)